operations/network/gateway_testing.md

```diff
@@ -9,20 +9,30 @@ will obtain an API token from Keycloak and then use that token to attempt to acc
 more networks, as defined in the gateway test definition file (`gateway-test-defn.yaml`). The test will check the
 return code to make sure it gets the expected response.
 
-When the `nmnlb` network is specified, the test will use `api-gw-service-nmn.local` as an override for
+When the `nmnlb` network is tested, the test will use `api-gw-service-nmn.local` as an override for
 `nmnlb.<system-domain>` in CSM v1.2. Optionally, setting `use-api-gw-override: false` in `gateway-test-defn.yaml`
 disables that override, and the test will use `nmnlb.<system-domain>`.
 
-## Running gateway tests on an NCN
+## Topics
+
+- [Gateway Testing](#gateway-testing)
+  - [Topics:](#topics)
+  - [Running gateway tests on an NCN management node](#running-gateway-tests-on-an-ncn-management-node)
```

Comment:
```suggestion
* [Running Gateway Tests on an NCN Management Node](#running-gateway-tests-on-an-ncn-management-node)
```

Reply:
I'll fix the capitalization, but style-checker requires `-` for unordered lists.

Reply:
I thought it was the other way around

Reply:
But I do see that the style checker isn't complaining about your PR currently on that score. But it likewise isn't complaining about my asterisks. So....weird.